lib/templates/_head.html

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="generator" content="dartdoc {{version}}">
```

Comment:
I love the friendly tone, too – but putting `>` (or even `<`) in tag attributes is a no-no.

...and `&gt;3` doesn't have the same feel :smile: